source/fundamentals/geo.txt

```diff
@@ -0,0 +1,271 @@
+.. _csharp-geo:
+
+===================
+Search Geospatially
+===================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, coordinates, location, geographic
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to work with **geospatial data** formats,
+indexes, and queries.
+
+Geospatial data represents a geographic location on the surface of the Earth.
+
+Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+- Points on a graph
+
+Geospatial Data Formats
+-----------------------
+
+All geospatial data in MongoDB is stored in one of the following formats:
+
+- GeoJSON, a format that represents geospatial data on an earth-like
+  sphere.
+
+- Legacy coordinate pairs, a format that represents geospatial data
+  on a Euclidean plane.
+
+GeoJSON
+~~~~~~~
+
+Use GeoJSON to store data that represents geospatial information on
+an earth-like sphere. GeoJSON is composed of one or more **positions**
+and a **type**.
+
+Positions
+`````````
+
+A position represents a single location and exists in code as an array
+containing the following values:
+
+- Longitude in the first position (required)
+- Latitude in the second position (required)
+- Elevation in the third position (optional)
+
+The following is the position of the MongoDB Headquarters in New York City, NY.
+
+.. code-block:: csharp
+
+   GeoJson.Position(-73.986805, 40.7620853)
+
+.. important:: Longitude then Latitude
+
+  GeoJSON orders coordinates with longitude first and latitude second.
```

Comment:
Consider also adding` GeoJson.Geographic(-73.986805, 40.7620853)` example, with ` GeoJson.Geographic` geographic coordinates names are used: longitude and longitude

Reply:
Could you provide a little more detail on this? A little unsure on how you would incorporate this example (also you say "longitude and longitude" here).

Reply:
Sorry, lon and lon is a typo;

There is a method in our `GeoJson` class, `GeoJson.Geographic` for constructing coordinates with Longitude and Latitude properties. It's equivalent, to Position, just different naming for clarity. We could mentioned that as well.